server/tasks.js

```diff
@@ -0,0 +1,40 @@
+
+export function scheduleTask(func, timeout, ...args) {
+    if (typeof func !== 'function')
+        throw new Error('Task to schedule must be a function')
+    if (func.toString().includes('[native code]'))
+        throw new Error('Task function contains native code.  Scheduled tasks must be serializable')
+    const id = ulid()
+    const timeoutInfo = setTimeout(() => {
+        func(...args)
```

Comment:
Does this run the risk that if func() crashes, the task won't get cleared?

Reply:
Yes, you're right, that would be better!  I will make it so.